ce/artifacts/artifact.ts

```diff
@@ -209,43 +204,71 @@ export class Artifact extends ArtifactBase {
     await this.targetLocation.delete({ recursive: true, useTrash: false });
   }
 
+  matchFilesInArtifact(glob: string) {
+    const results = match(this.allPaths, glob.trim(), { dot: true, cwd: this.targetLocation.fsPath, unescape: true });
```

Comment:
```suggestion
    const results = match(this.allPaths, glob, { dot: true, cwd: this.targetLocation.fsPath, unescape: true });
```

Do we really need this trim? If we do that sounds like a bug.

Reply:
Well, if someone has spaces around the glob in the artifact, `"{ **/*.h  }"` then you get spaces around the string.